### PR TITLE
Updates for observability after live review

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/4621#issuecomment-718257438

### CODE REVIEW

File: observability/manage_observe.adoc [hunk 1]
```diff
@@ -106,10 +106,10 @@ View the data from your managed clusters by accessing Grafana. Complete the foll
 You can also  access Grafana dashboards from the _Clusters_ page. From the navigation menu, select *Automate infrastructure* > *Clusters* > *Grafana*.
 . Access the Prometheus metric explorer by selecting the *Explore* icon from the Grafana navigation menu.
 
-[#disable-observability]
-== Disable observability 
+[#disable-metrics-collector]
+== Disable _metrics-collector_
```
Comment (bjoydeep) [inline observability/manage_observe.adoc:111]: Note: This change affects all managed clusters enabled for Observability. By default, all managed clusters that are OpenShift 4.*  and with labels `vendor: OpenShift` are enabled.

File: observability/manage_observe.adoc [hunk 2]
```diff
@@ -122,7 +122,7 @@ spec:
     enableMetrics: false #indicates the observability addon push metrics to hub server
 ----
 
-When you disable observability, the `multicluster-observability-operator` continues to run on the managed cluster, while the `metrics-collector` deployment is scaled to zero. When a managed cluster with the observability component is detached, the `metrics-collector` deployments are removed.
+*Note*: When a managed cluster with the observability component is detached, the `metrics-collector` deployments are removed.
```
Comment (bjoydeep) [inline observability/manage_observe.adoc:130]: Add another section here:
**Disable Managed Clusters from collecting Metrics at all**

As explained earlier, all managed clusters that are OpenShift 4.* and with labels vendor: OpenShift are enabled for metrics collection. If you want to disable certain managed clusters, you can do either of these:

1. Add a label to the Custom resource instance for CRD managedclusters.cluster.open-cluster-management.io - `observability: disabled`
2. Navigate to ACM UI Cluster overview page and add label `observability:disabled ` to the clusters we want to exclude

File: observability/observability_install.adoc [hunk 3]
```diff
@@ -120,7 +88,7 @@ spec:
     statefulSetStorageClass:gp2
 ----
 +
```
Comment (bjoydeep) [inline observability/observability_install.adoc:109]: The above default should work for most cases except:

1. `statefulSetStorageClass`: check the default storage class and update it
2. `statefulSetSize`: depending on the number of managed clusters, the number of resources per managed clusters and value of `retentionResolutionRaw` the size of the StatefulSets may need to changed

File: observability/observability_install.adoc [hunk 4]
```diff
@@ -120,81 +107,39 @@ spec:
...
-    reason: Disabled
-    status: "False"
-    type: Disabled
-  - lastTransitionTime: "2020-10-22T14:10:55Z"
-    message: Metrics collector deployment not successful
-    reason: Degraded
-    status: "False"
-    type: Degraded
-----
+*Note*: If you want to exclude specific managed clusters from collecting the observability data, add the following cluster label to your clusters: `vendor: OpenShift`.
```
Comment (dockerymick) [inline observability/observability_install.adoc:125]: @randymgeorge tagging you so that we remember to revisit this line

**Comment by dockerymick** [comment]
Made a comment on the issue here: https://github.com/open-cluster-management/backlog/issues/4621#issuecomment-719588951. Continuing to merge

**Pull request merged**